<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>item</class>
 <widget class="QDialog" name="item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>42</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>199</width>
    <height>42</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>199</width>
    <height>42</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_msg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>201</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
